### ISSUE TITLE
Sign in successful, auth.signedIn works, but auth.user and auth.authToken null

### ISSUE THREAD

**Issue by marbemac** [heartsentwined/ember-auth#123]
Also, there is no request post sign in to grab the user details. I'm using this in a expressjs app using brunch (http://brunch.io/). I'm using ember data. See the attached screenshot for an example of what's happening. Notice that it's also not including an authToken on the subsequent request. However, handlebars templates DO respond to auth.signedIn correctly. Any idea on how to start debugging this? I have ember auth working in a separate rails app. Thanks! I've really been enjoying ember auth.

![screen shot 2013-11-02 at 12 13 58 am](https://f.cloud.github.com/assets/847542/1458663/955360dc-438e-11e3-81ef-8185d08e5e87.png)

**Comment by marbemac** [comment]
Hi, I've created a simple example so you can reproduce the issue. The repo is here:

https://github.com/marbemac/ember-auth-test/tree/master

Instructions to run it are in the readme. It's just a basic expressjs app with brunch.

**Comment by scottfister** [comment]
Yeah, also finding this, even using the `custom` session shows that the `store` hook to save the token is never called.
